sim/warrior/arms/talents.go

```diff
@@ -50,6 +50,7 @@ func (war *ArmsWarrior) applyTasteForBlood() {
 		ActionID: actionID,
 		Duration: time.Second * 9,
 		Tag:      warrior.EnableOverpowerTag,
+		Icd:      &core.Cooldown{Duration: time.Second * 6},
```

Comment:
this ICD is doing nothing atm. In the handler for activating this aura you need to check if the ICD is ready then activate and use the ICD

Reply:
Yeah, this is only for tracking the auras cooldown but required the SpellAPLFlag on the Trigger to use.  Removiung till I figure something better out.